src/App.tsx

```diff
@@ -8,7 +7,57 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [actualTodos, setActualTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [query, setQuery] = useState('');
+  const [statusSelect, setStatusSelect] = useState('all');
+
+  const onSelectedTodo = (todoId: number) => {
+    setSelectedTodo(todos.find(todo => todo.id === todoId) || null);
+  };
+
+  const setClose = (item: null) => {
+    setSelectedTodo(item);
+  };
```

Comment:
```suggestion
  const handleModalClose = (item: null) => {
    setSelectedTodo(item);
  };
```
not description name